packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -526,6 +532,77 @@ test.serial('put() does not write to Dynamo/Elasticsearch or publish SNS message
   t.is(Messages, undefined);
 });
 
+test.serial('put() does not write to Dynamo/Elasticsearch or publish SNS message if writing to PostgreSQL fails and no Dynamo record existed previously', async (t) => {
```

Comment:
I'm not sure that I understand why this test is needed in addition to `'put() does not write to Dynamo/Elasticsearch or publish SNS message if writing to PostgreSQL fails'`

Reply:
Fair, the implementation here doesn't really lead to the 'why'. 

The intent here is to cover the new branch logic 

```
    if (oldCollection) {
      await collectionsModel.create(oldCollection);
    }
```

in that if that branch wasn't present, this would *throw* as oldCollection wouldn't be defined. 

That test could possibly be named better, or implemented more elegantly 🤔

Reply:
I see. I don't think the test needs to change then. Just wanted to understand the reason for it